web-ui/src/pages/OnboardProgressPage.jsx

```diff
@@ -36,23 +33,17 @@ export default function OnboardProgressPage(onboardee){
   const [open, setOpen] = useState(false);
   //const [empFile, setEmpFile] = useState(" ");
   //const [offer, setOfferFile] = useState(" ");
-  const [addOnboardeeModal, setAddOnboardeeModal] = useState(false);
   const { state, dispatch } = useContext(AppContext);
   const { csrf , onboardeeProfiles} = state;
-
+  const handleAddModalClose = () => setOpen(false);
   const handleOpen = () => setOpen(true);
-  const handleClose = () => {
-    setOpen(false);
-    //setEmpFile(" ");
-    //setOfferFile(" ");
-  };
   // const handleSubmitClose = () => {
   //   setOpen(false);
   //   setAddOnboardeeModal(true);
   // };
-  const handleMsgModalClose = () => {
-    setAddOnboardeeModal(false);
-  };
+  // const handleMsgModalClose = () => {
```

Comment:
It looks like this commented code is still present, let's delete lines 40-49 unless needed.